gridwars.bmx

```diff
@@ -2597,7 +2598,7 @@ Type pu
 		disty# = y-py
 		dist# = (distx * distx + disty * disty)
 		If dist < 20*20 + 16*16
-			PowerUp()
+			UpPower()
```

Comment:
I'm not an expert in BlitzMax, could you tell why did this change fix the build?

Reply:
There is a global variable (control.bmx:L31)
```
Global POWERUP:Int = 2500
```
It looks like BMX-ng compiler mixes it up with function name PowerUp(). Effectively it's trying to substitute function call with variable value. I'm not a great BMX expert too. I assume the modern compiler ignores case of all identifiers, so PowerUp becomes POWERUP internally. Solution was to change function name so that it not interfere with name of variable.